apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -133,9 +133,9 @@ def add_edges_for_groups(
         )
 
     # if a group G depends on a node N, all blocks in the group G should depend on Node N
-    for group_tag in groups:
+    for group_tag, nodes in groups.items():
         for dependency, _ in graph.in_edges(group_tag):
-            for node in groups[group_tag]:
+            for node in nodes:
                 graph.add_edge(dependency, node)
```

Comment:
why is this change in this PR?

Reply:
All the other rules I inverted had multiple changes that I felt warranted a separate PR; but this rule violation had just the single change.

Reply:
I can split it out still if wanted.

Reply:
Going to resolve this; I am quite sure this is not breaking and I have updated the title to avoid any potential confusion.

Reply:
that's fine with me